packages/cli/oas/medusa-oas-cli/src/command-oas.ts

```diff
@@ -161,7 +161,7 @@ async function getOASFromCodebase(
 async function getPublicOas(
   apiType: ApiType,
 ) {
-  const url = `https://docs.medusajs.com/v2/api/download/${apiType}`
+  const url = `https://docs.medusajs.com/v2/api/api/download/${apiType}`
```

Comment:
Due to how the API reference is built, we have to use a base path `/v2/api` to have routes like `/v2/api/store` and `/v2/api/admin`, which means actual APIs in the project are under `/v2/api/api`.

This setup wasn't in the v1 API reference because it was the main proxy project (i.e. no other projects redirecting to it based on the page's path). For the v2 API reference, using just a `/v2` base path causes redirect and 404 errors when trying to fetch its static files (CSS, js, etc...) through the proxy (as it tries to fetch these files from other doc projects). So, it has to be under a `/v2/api` base path.